src/main/kotlin/lotto/domain/value/LottoNumber.kt

```diff
@@ -0,0 +1,11 @@
+package lotto.domain.value
+
+data class LottoNumber(private val number: Int) {
+    init {
+        if (number !in 1..45) throw IllegalArgumentException("로또는 1~45사이의 숫자만 생성가능합니다.")
```

Comment:
`require()`는 값을 만족하지 않으면 `IllegalArgumentException`을 발생시킵니다.

```suggestion
        require(number in 1..45) { "로또는 1~45사이의 숫자만 생성가능합니다." }
```